boards/arm/raytac_mdbt50q_db_40_nrf52840/board.cmake

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=nrf52840" "--speed=4000")
+board_runner_args(pyocd "--target=nrf52840" "--frequency=4000000")
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/openocd-nrf5.board.cmake)
```

Comment:
328KB is far too large for an image, please vastly reduce width and height of the image and then use https://imagecompressor.com/ to shrink the size down

Reply:
Hi Jamie,
Thank you for your reminder. I will correct the problem you mentioned and upload it again.